### PR TITLE
Add distribute_from_whales real time functionality

### LINKED ISSUE #5: Allow using `development` chain in Network manager
Allow its use on the development environment since there might be a case in which the gas performance of certain actions will be assessed. Currently failing with this error when attempting to use the network manager while on development:

```strategies.set_default_for_active_chain()
  File "<console>", line 1, in <module>
  File "./badger_utils/gas_utils/gas_utils.py", line 55, in set_default_for_active_chain
    chain = network_manager.get_active_network()
  File "./badger_utils/network_manager.py", line 40, in get_active_network
    raise Exception(f"Chain ID {active_network} not recognized")
Exception: Chain ID development not recognized```

### PR DESCRIPTION
**Pull request by SHAKOTN**
Implements #7 
Additionally fixes #5 
**Reasoning**:
Locally defined whales are outdated and have mismatches with real data on-chain

**Solution**:
Fetch whales in real time from [Ethplorer API](https://github.com/EverexIO/Ethplorer/wiki/Ethplorer-API#get-top-token-holders) and distribute tokens from forked chain

**Done:**
1. Add separate module `distribute_from_whales_realtime`
2. Add function `is_address_eoa` to check if address is eoa. We don't want to distribute from "whale" contracts, only from eoa
3. Add `ehtplorer_utils` module to interact with ethplorer API
4. Lots and lots of taste to cover this all

### CODE REVIEW

File: badger_utils/constants.py [hunk 1]
```diff
@@ -151,3 +151,32 @@ class ContractSystems(Enum):
...
+    "0x6dEf55d2e18486B9dDfaA075bc4e4EE0B28c1545",
+    "0x64eda51d3Ad40D56b9dFc5554E06F94e1Dd786Fd",
+    "0xb9D076fDe463dbc9f915E5392F807315Bf940334",
+    "0xb19059ebb43466C323583928285a49f558E572Fd",
+    "0xDE5331AC4B3630f94853Ff322B66407e0D6331E8",
+    "0x2fE94ea3d5d4a175184081439753DE15AeF9d614",
+    "0x410e3E86ef427e30B9235497143881f717d93c2A",
+    "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
+    "0x110492b31c59716AC47337E616804E3E3AdC0b4a",
+    "0x110492b31c59716AC47337E616804E3E3AdC0b4a",
```
Comment (sajanrajdev) [inline badger_utils/constants.py:173]: This token is repeated on line 172

**Comment by sajanrajdev** [comment]
@SHAKOTN, I attempted to import the `distribute_from_whales_realtime` function on the fork environment, as we will always do with this function, and it returned the same error seen before regarding the fork being un-recognized as a chain:

image

This is a good opportunity to fix this issue,. To replicate:
- `brownie console`
- >>> `from badger_utils.token_utils.distribute_from_whales_realtime import distribute_from_whales_realtime`

File: badger_utils/constants.py (hunk 1 above)
Comment (SHAKOTN) [reply badger_utils/constants.py:173]: True. Copypaste issue

File: badger_utils/constants.py [hunk 2]
```diff
@@ -151,3 +151,32 @@ class ContractSystems(Enum):
...
+    "0xCEfF51756c56CeFFCA006cD410B03FFC46dd3a58",
+    "0x758A43EE2BFf8230eeb784879CdcFF4828F2544D",
+    "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+    "0x798D1bE841a82a273720CE31c822C61a67a601C3",
+    "0xeb4c2781e4eba804ce9a9803c67d0893436bb27d",
+    "0xcA3d75aC011BF5aD07a98d02f18225F9bD9A6BDF",
+    "0x4e3fbd56cd56c3e72c1403e103b45db9da5b9d2b",
+    "0x62b9c7356a2dc64a1969e19c23e4f579f9810aa7",
```
[the remark this answers is not in the document]
Comment (SHAKOTN) [reply badger_utils/constants.py:180]: This is fixed now

File: badger_utils/constants.py (hunk 1 above)
Comment (SHAKOTN) [reply badger_utils/constants.py]: Fixed

**Comment by SHAKOTN** [comment]
> * brownie console

Alright, will check it now

**Comment by SHAKOTN** [comment]
> @SHAKOTN, I attempted to import the `distribute_from_whales_realtime` function on the fork environment, as we will always do with this function, and it returned the same error seen before regarding the fork being un-recognized as a chain:
> 
> image
> 
> This is a good opportunity to fix this issue,. To replicate:
> 
> * `brownie console`
> * `from badger_utils.token_utils.distribute_from_whales_realtime import distribute_from_whales_realtime`

Fixed

**Comment by sajanrajdev** [comment]
Got the following error when running the `distribute_from_whales_realtime` function while it processed the 4th token from the list: `0xcD7989894bc033581532D2cd88Da5db0A4b12859`. Did you encountered this? I don't think it is related to the token but rather to the rpc call but can't check in detail right now.

image
...
image

Also, my apologies, we really don't need to run these functions in development but rather on fork which works well. I am just not used to having the dev environment as the default so when I ran `brownie console` I assumed it was running mainnet-fork as usual but it was development, which is fine.

I will remember to test these functions using `brownie console --network mainnet-fork` from now and on as this is the environment where tests are executed in the other repos.

**Comment by sajanrajdev** [comment]
Tested both `distribute_from_whale_realtime` and `distribute_from_whales_realtime` and they worked like a charm.

image

A little nit suggestion is that I would align the order of parameters of these two functions for ease of use: token, receiver, %. Otherwise it LGTM!

**Comment by sajanrajdev** [comment]
Tested functions with parameters order change and it worked perfectly. LGTM
image

**Review by sajanrajdev — APPROVED** [review]

**Pull request merged**